### PR TITLE
Redux Tools Documentation

### PR DESCRIPTION
**Pull request by dunaevskiy**
Reworked documentation of Redux Tools.
There also will be a few code examples (later).

**How to set up**
`GitHub Settings` -> `GitHub Pages` -> Source `master branch /docs folder`

**Local development**
```bash
yarn global add docsify
docsify serve ./docs
```

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+   -  [Middleware (React Extension)](/middleware-react/index.md)
+   -  [Thunk (Extended)](/thunk/index.md)
+
+- Internal packages
+
+   -  [Namespaces](/namespaces/index.md)
+   -  [Injectors](/injectors/index.md)
+   -  [Injectors (React Extension)](/injectors-react/index.md)
+   -  [Utils](/utils/index.md)
+   -  [Utils (React Extension)](/utils-react/index.md)
```
Comment (wafflepie) [inline docs/_sidebar.md]: React Utils

File: docs/actions/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	makeReducer,
+} from '@redux-tools/actions';
+
+export const ActionTypes = makeActionTypes(
+	'duck',
+	['ADD', 'ADD_CRAZY', 'INCREMENT', 'REMOVE'],
+);
+
+export const add = makeSimpleActionCreator(ActionTypes.ADD);
+export const addCrazy = makeActionCreator(ActionTypes.CRAZY_ADD, 2, null);
```
Comment (wafflepie) [inline docs/actions/index.md]: ADD_CRAZY (typo), also, the usage of makeActionCreator is wrong.

File: docs/actions/index.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+1. `type` ( _string_ ): The action type
+2. `getPayload` ( _any –> any_ ): Payload getter
+3. `getMeta` ( _any -> any_ ): Meta getter
+
+**Returns**
+
+( _any -> Action_ ): An action creator
+
+### makeBinaryActionCreator()
```
Comment (wafflepie) [inline docs/actions/index.md]: i don't think this is yet implemented 
https://github.com/lundegaard/redux-tools/issues/73

File: docs/actions/index.md (hunk 2 above)
Comment (dunaevskiy) [reply docs/actions/index.md]: > ADD_CRAZY (typo), also, the usage of makeActionCreator is wrong.
Typo - fixed, but not sure what do You mean ("the usage").

**Pull request merged**